project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16-M1
```

Comment:
@eed3si9n @hseeberger Setting the sbt version to 0.13.16-M1 will automatically cross-compile plugins for sbt 1.0.0? I'm asking because I can't see a reference to sbt 1.0.0.